### PR TITLE
Workaround to fix the camera positioning issue on the google map view

### PR DESCRIPTION
**Pull request by cyanglaz**
A temporary workaround to fix the camera positioning issue mentioned in https://github.com/flutter/flutter/issues/24806

### CODE REVIEW

File: packages/google_maps_flutter/ios/Classes/GoogleMapController.m [hunk 1]
```diff
@@ -217,6 +221,12 @@ - (void)mapView:(GMSMapView*)mapView willMove:(BOOL)gesture {
 }
 
 - (void)mapView:(GMSMapView*)mapView didChangeCameraPosition:(GMSCameraPosition*)position {
+  if (!_cameraDidInitialSetup) {
+    // A temporary workaround for a bug that the camera is not properly positioned at
```
Comment (amirh) [inline packages/google_maps_flutter/ios/Classes/GoogleMapController.m]: Note that we suspect that this is an issue with the iOS Google Maps SDK(which is still being investigated), and that we call moveCamera to the current position as a temporary workaround.
Also include a TODO to remove this once the Maps SDK issue is resolved(I'm ok with closing the current Github issue with this workaround, and filing a separate one for this TODO).

Comment (amirh) [reply packages/google_maps_flutter/ios/Classes/GoogleMapController.m]: I meant note it in the code comment 😄

Comment (cyanglaz) [reply packages/google_maps_flutter/ios/Classes/GoogleMapController.m]: updated the comments, and also opened the issue ticket.

File: packages/google_maps_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 0.2.0+1
+
+* Fixed a bug which the camera is not positioned correctly at map initialization(temporary workaround).
```
Comment (amirh) [inline packages/google_maps_flutter/CHANGELOG.md]: nit: we can reference the github issue.
I would also mentioned that it's a work around, e.g "Fixed(worked around) a bug..."

Comment (cyanglaz) [reply packages/google_maps_flutter/CHANGELOG.md]: I did add the `workaround` in the end of the sentence. Did you mean to remove it and add `Fixed(worked around)`?

Comment (amirh) [reply packages/google_maps_flutter/CHANGELOG.md]: ohh oops missed that, that looks good.

Comment (amirh) [reply packages/google_maps_flutter/CHANGELOG.md]: still lets reference the github issue

**Pull request merged**